tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -442,3 +442,42 @@ Open New Notebook In Jupyterlab Menu
     Close Other JupyterLab Tabs
     Maybe Close Popup
     Sleep  1
+
+Log In N Users To JupyterLab And Launch A Notebook For Each Of Them
+    [Documentation]    Log in N users and run notebook for each of them
+    [Arguments]    ${list_of_usernames}
+    FOR    ${username}    IN    @{list_of_usernames}
+        Open Browser    ${ODH_DASHBOARD_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}    alias=${username}
```

Comment:
Yes
there is valid reson
if we log in and log out for now it will give us a result write because at the end server is not stoped 
what if in the future they  turn of server when we log out
then it will fail because active user count will not be same
and another reson is if we want to run different activities on jupyter notebook this will easy